src/SonarQube.Client/Messages/RoslynExportProfileRequest.cs

```diff
@@ -27,8 +27,14 @@ public class RoslynExportProfileRequest
         [JsonProperty("language")]
         public string LanguageKey { get; set; }
         [JsonProperty("name")]
-        public string QualityProfileName { get; set; }
+        public virtual string QualityProfileName { get; set; }
         [JsonProperty("exporterKey", Required = Required.Always)]
         public string ExporterKey => $"roslyn-{LanguageKey}";
     }
+
+    public class RoslynExportProfileRequestV66Plus : RoslynExportProfileRequest
```

Comment:
That's not ideal but this is the easiest fix I could think of.

Reply:
Neat! Simpler than having to customise the JSON serializer.